dist/images/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN INSTALL_PKGS=" \
 	PyYAML bind-utils procps-ng openssl numactl-libs firewalld-filesystem \
 	libpcap kubectl \
 	iproute strace socat \
-	unbound-libs \
+	unbound-libs libibverbs \
```

Comment:
This isn't actually needed unless you are using infiniband cards. OVS complains loudly about it not being there but that's just OVS being stupid.

Reply:
We have an OVS bug open about it at RH FWIW. But yeah, we don't need to add this RPM.